policy/release/quay_expiration.rego

```diff
@@ -0,0 +1,55 @@
+#
+# METADATA
+# title: Labels
```

Comment:
This seems a bit ambiguous with the already existing [labels](https://enterprisecontract.dev/docs/ec-policies/release_policy.html#labels_package) package.

Reply:
Oops, copy paste snafu I think. Thanks!